pyteal/ast/subroutine_test.py

```diff
@@ -170,6 +187,7 @@ def fn_2arg_1ret_with_expr(
             ],
             "fn_2arg_1ret_with_expr",
             pt.abi.ByteTypeSpec(),
+            "fn_2arg_1ret_with_expr(byte[10])byte",  # TODO: this seems wrong, no?
```

Comment:
No way this is right, right? 

The first arg is an Int, not an abi type

Reply:
this leads back to the comment I had on `signature` method, which only applies to registrable method.